x/gov/client/cli/prompt_test.go

```diff
@@ -0,0 +1,43 @@
+package cli_test
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/chzyer/readline"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+)
+
+type st struct {
+	ToOverflow int
+}
+
+// On the tests running in Github actions, somehow there are races.
+var globalMu sync.Mutex
```

Comment:
This isn't used and why was it even added?

Reply:
This has been changed in https://github.com/cosmos/cosmos-sdk/pull/13350